packages/angular/projects/ui-angular/src/lib/components/authenticator/components/form-field/form-field.component.html

```diff
@@ -24,6 +24,7 @@
     [disabled]="disabled"
     [labelHidden]="labelHidden"
     [autocomplete]="inferAutocomplete()"
+    (setBlur)="onBlur($event)"
```

Comment:
Angular linter complained here if I called this `blur` or `onblur`. So I changed the name to `setBlur`